### PR TITLE
Update log level setting pattern

### PR DESCRIPTION
**Pull request by mshriver**
Small change to the log level setting pattern.

Needs testing, pushing to get early feedback on the approach.

### CODE REVIEW

File: broker/commands.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
...
 @click.group()
-@click.option("--debug", is_flag=True)
-def cli(debug):
-    if debug or settings.DEBUG:
-        b_log.setup_logzero(level="debug")
-    logger.debug("I'm in debug mode")
+@click.option("--log-level",
+              type=click.Choice(['info', 'warning', 'error', 'critical', 'debug']))
+def cli(log_level):
+    b_log.update_logging_level(level=log_level)  # the method handles uppercase
```
Comment (JacobCallahan) [inline broker/commands.py:15]: should you instead just call setup_logzero as before?

File: broker/logger.py [hunk 2]
```diff
@@ -6,27 +6,23 @@
...
+    formatter = logzero.LogFormatter(fmt=debug_fmt if log_level == logging.DEBUG else log_fmt)
     logzero.setup_default_logger(formatter=formatter)
     logzero.loglevel(level)
     logzero.logfile(
-        path, loglevel=level, maxBytes=1e9, backupCount=3, formatter=formatter
+        path, loglevel=log_level, maxBytes=1e9, backupCount=3, formatter=formatter
     )
 
 
+def update_logging_level(level="info"):
```
Comment (JacobCallahan) [inline broker/logger.py:23]: if you call setup_logzero, then this function wouldn't be needed

File: broker/commands.py (hunk 1 above)
Comment (mshriver) [reply broker/commands.py:15]: Responding here as both comments are related.

Correct, but I considered this as a specific change to the log configuration that could be called separately.

In reviewing though I see that this won't end up updating the log formatting.

I'll switch up to re-call existing method, so the formatting gets updated

File: broker/commands.py [hunk 3]
```diff
@@ -8,12 +9,23 @@
...
+
+
 @click.group()
-@click.option("--debug", is_flag=True)
-def cli(debug):
-    if debug or settings.DEBUG:
-        b_log.setup_logzero(level="debug")
-    logger.debug("I'm in debug mode")
+@click.option(
+    "--log",
```
Comment (JacobCallahan) [inline broker/commands.py:20]: a bit of a nitpick, but I think ```--log-level``` is a better option, as I would consider ```--log``` to potentially be a log path.

Comment (mshriver) [reply broker/commands.py:20]: You know I spent several minutes debating internally whether or not it was better shortened. :)

**Pull request merged**